Servers/utils/question.utils.ts

```
@@ -100,63 +119,94 @@ export const addFileToQuestion = async (
+  });
+
+  return result[0];
 };
 
 export const deleteQuestionByIdQuery = async (
   id: number
-): Promise<Question | null> => {
-  const result = await pool.query(
-    "DELETE FROM questions WHERE id = $1 RETURNING *",
-    [id]
+): Promise<Boolean> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)